src/deps.ts

```diff
@@ -1,21 +1,17 @@
-export { assert, assertEquals, assertRejects, assertThrows } from 'https://deno.land/std@0.141.0/testing/asserts.ts'
-export * as base64 from 'https://deno.land/std@0.141.0/encoding/base64.ts'
-export * as flags from 'https://deno.land/std@0.141.0/flags/mod.ts'
-export * as colors from "https://deno.land/std@0.141.0/fmt/colors.ts"
-export * as fs from 'https://deno.land/std@0.141.0/fs/mod.ts'
-export * as path from 'https://deno.land/std@0.141.0/path/mod.ts'
-export * as streams from "https://deno.land/std@0.141.0/streams/mod.ts"
-export { default as tweetnacl } from 'https://esm.sh/tweetnacl@1.0.3?pin=v120'
-export { base58btc } from 'https://esm.sh/multiformats@11.0.2/bases/base58?pin=v120'
-export { type Multibase } from 'https://esm.sh/multiformats@11.0.2?pin=v120'
-export { default as blake } from 'https://esm.sh/@multiformats/blake2@1.0.13?pin=v120'
-export { CID } from 'https://esm.sh/multiformats@11.0.2/cid?pin=v120'
-export { miniexec as sh } from "https://deno.land/x/miniexec@1.0.0/mod.ts"
-export * as esbuild from "https://deno.land/x/esbuild@v0.14.47/mod.js"
-export * as sqlite from "https://deno.land/x/sqlite@v3.7.1/mod.ts"
-export { type DB as SQLiteDB, type PreparedQuery } from "https://deno.land/x/sqlite@v3.7.1/mod.ts"
-// Other third-party shell libs:
-// https://deno.land/x/shell_tag
-// https://deno.land/x/violet
-// https://deno.land/x/dzx
-// https://deno.land/x/miniexec
+export { assert, assertEquals, assertRejects, assertThrows } from 'jsr:@std/assert@1.0.13';
+export * as base64 from 'jsr:@std/encoding@1.0.10/base64';
+export * as flags from 'jsr:@std/flags@0.224.0';
+export * as colors from 'jsr:@std/fmt@1.0.8/colors';
+export * as fs from 'jsr:@std/fs@1.0.19';
+export * as path from 'jsr:@std/path@1.1.1';
+export * as streams from 'jsr:@std/streams@1.0.10';
+export * as util from 'jsr:@std/io@0.225.2';
+export { copy, readAll } from 'jsr:@std/io@0.225.2';
+export * as sqlite from "jsr:@db/sqlite@0.12.0";
+export { type Database as SQLiteDB } from "jsr:@db/sqlite@0.12.0";
+export * as esbuild from "https://deno.land/x/esbuild@v0.25.6/mod.js";
```

Comment:
This is the last deno.land dependency. Could you grab it from npm instead? (remembering to specify precise version)

Reply:
https://github.com/okTurtles/chel/pull/46/commits/41c58ec788acef6e841c9153bbe246d11947903f